File: 251122/윤년인가/is-leap-year.java
import java.util.Scanner;

public class Main {
    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);
        int year = sc.nextInt();
        boolean isLeapYear = (year % 400 == 0) || (year % 4 == 0 && year % 100 != 0);
        System.out.println(isLeapYear);
    }
}

File: 260101/윤년인가/is-leap-year.kt
import java.io.BufferedReader
import java.io.InputStreamReader

fun main() = with(BufferedReader(InputStreamReader(System.`in`))) {
    val year = readLine().toInt()
    val isLeapYear = (year % 4 == 0 && year % 100 != 0) || (year % 400 == 0)
    println(isLeapYear)
}